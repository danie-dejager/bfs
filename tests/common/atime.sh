cd "$TEST"

now=$(epoch_time)

"$XTOUCH" -at "@$((now - 60 * 60 * 24 * 7))" last_week
"$XTOUCH" -at "@$((now - 60 * 60 * 49))" two_days_ago
"$XTOUCH" -at "@$((now - 60 * 60 * 25))" yesterday
"$XTOUCH" -at "@$((now - 60 * 60))" one_hour_ago
"$XTOUCH" -at "@$((now))" now
"$XTOUCH" -at "@$((now + 60 * 60 * 24))" tomorrow

bfs_diff . -mindepth 1 \
    \( -atime -1 -exec printf -- '-atime -1: %s\n' {} \; -o -true \) \
    \( -atime  1 -exec printf -- '-atime  1: %s\n' {} \; -o -true \) \
    \( -atime +1 -exec printf -- '-atime +1: %s\n' {} \; -o -true \)
