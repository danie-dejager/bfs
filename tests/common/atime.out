-atime  1: ./yesterday
-atime +1: ./last_week
-atime +1: ./two_days_ago
-atime -1: ./now
-atime -1: ./one_hour_ago
-atime -1: ./tomorrow
